conf4j-core/src/main/java/com/sabre/oss/conf4j/converter/DefaultTypeConverters.java

```diff
@@ -25,7 +25,22 @@
 package com.sabre.oss.conf4j.converter;
 
 import com.sabre.oss.conf4j.converter.composite.JsonLikeTypeConverter;
-import com.sabre.oss.conf4j.converter.standard.*;
+import com.sabre.oss.conf4j.converter.standard.BigDecimalTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.BooleanTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.ByteTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.CharacterTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.DoubleTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.DurationTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.EnumTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.EscapingStringTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.FloatTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.IntegerTypeConverter;
+import com.sabre.oss.conf4j.converter.standard.LocalDateTimeTypeConverter;
```

Comment:
In this project, when there is 5+ classes imported with the same package, '*' is used. Could review you changes and fix import section accordingly?